Dockerfile

```diff
@@ -9,10 +9,10 @@ RUN go mod download
 COPY . .
 ARG TARGETOS TARGETARCH TARGETVARIANT
 RUN \
-    if [ "${TARGETARCH}" = "arm" ] && [ -n "${TARGETVARIANT}" ]; then \
-      export GOARM="${TARGETVARIANT#v}"; \
-    fi; \
-    GOOS=${TARGETOS} GOARCH=${TARGETARCH} CGO_ENABLED=0 go build -v ./cmd/golink
+  if [ "${TARGETARCH}" = "arm" ] && [ -n "${TARGETVARIANT}" ]; then \
```

Comment:
I don't see a non-whitespace diff here, is this meant to be part of the change?

Reply:
it wasn't exactly intentional, but got autoformatted.  I do actually prefer it this way, so think I'll leave it.